geometry_msgs/PoseStamped initial_pose
---
bool success
---
float32 distance_error